test/runtests.jl

```diff
@@ -17,6 +17,24 @@ test_show(lm1)
 @test_approx_eq vcov(lm1) Σ
 @test_approx_eq cor(lm1.model) diagm(diag(Σ))^(-1/2)*Σ*diagm(diag(Σ))^(-1/2)
 
+#1).
+data=DataFrame(X=[1,2,3],Y=[2,4,7])
```

Comment:
On master the tests include many other values like fit statistics. I'd say better leave this for another PR.